### PR TITLE
Copy files from To-do-list

### PR DESCRIPTION
**Pull request by Lornakaboro**
## In this activity I copied code from To-Do list

### CODE REVIEW

File: modules/displayList.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    this.getTasksFromLocalStorage();
+  }
+
+  updateTasksInLocalStorage() {
+    localStorage.setItem('taskList', JSON.stringify(this.listArray));
+  }
+
+  getTasksFromLocalStorage() {
+    this.listArray = JSON.parse(localStorage.getItem('taskList')) || [];
+  }
```
Comment (zdnahom) [inline modules/displayList.js]: Kindly use the ES6 arrow function instead of normal functions. To make your code clean.

**Review by zdnahom — CHANGES_REQUESTED** [review]
Hey @Lornakaboro 👋 
Good Job so far!
There are still some issues left to address but you are almost there👍 


## Highlights
✔️ Linter passed
✔️ Great PR description
✔️ Descriptive commits

## Required Changes ♻️
Check the comments under the review.
## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
<b>Please, remember to tag me in your question so I can receive the notification.<b>

File: modules/displayList.js (hunk 1 above)
Comment (Lornakaboro) [reply modules/displayList.js]: Thankyou Nahom for the amazing feedback 👍

**Review by zdnahom — APPROVED** [review]
## Status: Approved ✔️

Hi  @Lornakaboro ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

## Highlights
✔️ Linter passed
✔️ Great PR description
✔️ Descriptive commits
✔️ Used ES6 best practice

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

<b>Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.<b>

**Pull request merged**